### PR TITLE
refactor : remove tools.h

### LINKED ISSUE #648: File "source/src_pw/tools.h" will not be used in ABACUS development planning.
## Describe Current Status and Possible Solution

In ABACUS code, there are so many files included header "src_pw/tools.h", which contains lots of header files in "source/module_base/".

We plan to remove it, and won't use this file any more.

### PR DESCRIPTION
**Pull request by dyzheng**

### CODE REVIEW

**Comment by caic99** [comment]
I suggest not using relative path for header files.

**Review by caic99 — APPROVED** [review]

**Comment by dyzheng** [comment]
> I suggest not using relative path for header files.

We should discuss the standards of relative path.

File: source/module_cell/unitcell.cpp [hunk 1]
```diff
@@ -58,7 +58,7 @@ UnitCell::~UnitCell()
 	delete[] lc;
 }
 
-
+#include "../src_parallel/parallel_common.h"
```
Comment (caic99) [inline source/module_cell/unitcell.cpp:63]: Shall this line wrapped in `#ifdef __MPI`?

File: source/module_md/FIRE.cpp [hunk 2]
```diff
@@ -1,5 +1,8 @@
 #include "FIRE.h"
 #include "MD_func.h"
+#ifdef __MPI
+#include <mpi.h>
```
Comment (caic99) [inline source/module_md/FIRE.cpp]: Better use "mpi.h" here.

Comment (dyzheng) [reply source/module_md/FIRE.cpp]: Done.

File: source/module_cell/unitcell.cpp (hunk 1 above)
Comment (dyzheng) [reply source/module_cell/unitcell.cpp:63]: No

**Pull request merged**